src/Mod/Part/App/TopoShape.cpp

```diff
@@ -2503,6 +2503,62 @@ TopoDS_Shape TopoShape::makeLongHelix(Standard_Real pitch, Standard_Real height,
     return TopoDS_Shape(std::move(wire));
 }
 
+TopoDS_Shape TopoShape::makeSpiralHelix(Standard_Real radiusbottom, Standard_Real radiustop,
+                                  Standard_Real height, Standard_Real nbturns,
+                                  Standard_Real breakperiod, Standard_Boolean leftHanded) const
+{
+    // 1000 periods is an OCCT limit. The 3D curve gets truncated
+    // if he 2D curve spans beyond this limit.

```

Comment:
@tomate44 do you mind `s/he/the/` on this line?

Reply:
Yes, "the", thanks